MediaBrowser.Providers/Tmdb/TmdbUtils.cs

```diff
@@ -14,6 +14,12 @@ public static class TmdbUtils
 
         public static string MapCrewToPersonType(Crew crew)
         {
+            if (crew.Department.Equals("production", StringComparison.InvariantCultureIgnoreCase)
+                && crew.Job.IndexOf("director", StringComparison.InvariantCultureIgnoreCase) != -1)
+            {
+                return PersonType.Director;
+            }
+
```

Comment:
I'm not sure if this is necessary since it seemed to find the series' director just fine without it, but it seemed weird to omit it from the mapper.